WireUI/Sources/WireConversationUI/ConversationTitleView.swift

```diff
@@ -32,6 +32,8 @@ public class ConversationTitleView: UIView {
     private let dropdownImage = UIImageView(image: .dropdown)
     private let tapButton = UIButton(type: .custom)
 
+    var accentColor: UIColor = .white
```

Comment:
suggestion: add a `didSet` handler and make `updateAccentColor(_:)` private.
```suggestion
    var accentColor: UIColor = .white {
        didSet { updateAccentColor(accentColor) }
    }
```

Reply:
Now with accent color discussions, there is no not to store accent color here